src/file_pipe_log/pipe.rs

```diff
@@ -401,15 +413,11 @@ impl<F: FileSystem> SinglePipe<F> {
         Ok(handle)
     }
 
-    fn maybe_sync(&self, force: bool) -> Result<()> {
+    fn sync(&self) -> Result<()> {
```

Comment:
In current implementation of `sync`, it still gets the `mutex` for avoiding concurrent racing at the beginning.

According to #258 , will it be refactored to support concurrent accessing in later work ? i.e. replace `mutex` with `Rwlock` ? or add an extra `F::Handle` in `SinglePipe`, making it be called concurrently ?

Reply:
Yes.